src/extensions/ignite/add-module.ts

```diff
@@ -1,40 +1,39 @@
-const { trim } = require('ramda')
-const path = require('path')
+import { trim } from 'ramda'
+import * as path from 'path'
+import { IgniteToolbox } from '../../types'
 
-module.exports = (plugin, command, context) => {
-  const getModuleName = (moduleName, options) => {
+export type AddModuleOptions = {
+  link?: boolean
+  dev?: boolean
+  version?: string
+}
+
+export default (toolbox: IgniteToolbox) => {
+  const getModuleName = (moduleName, options: AddModuleOptions) => {
     let name
     if (options.version) {
       name = `${moduleName}@${options.version}`
-    } else if (context.parameters.second.includes(path.sep)) {
+    } else if (toolbox.parameters.first.includes(path.sep)) {
```

Comment:
Again switching from `second` to `first`. Just checking if that's expected.

Reply:
Yessir! Gluegun 0.x was a mess. Gluegun 3.x is a slightly better mess.